### PR TITLE
Added some test to DC Visualization

### PR DESCRIPTION
**Pull request by ar4s**
Motivation: Code without tests is bad code.
Added way to test code written in JS by QUnit and PhantomJS.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -77,8 +77,9 @@ gulp.task('js', function(){
 });
```
Comment (pnuckowski) [inline gulpfile.js:77]: First line of this file should ends with comma, not semicolon.

Comment (ar4s) [inline gulpfile.js:77]: Thanks for review!

**Comment by mkurek** [comment]
is it still WIP? If not, it's :ok: for me

**Comment by szok** [comment]
:ok:

**Pull request merged**